<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDPSendDataClass</class>
 <widget class="QWidget" name="UDPSendDataClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDPSendData</string>
  </property>
  <widget class="QLabel" name="TargetIP_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TargetIP_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.20.201</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TargetPort_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>6501</string>
   </property>
  </widget>
  <widget class="QLabel" name="TargetPort_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="LocalPort_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Local Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="LocalIP_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Local IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LocalPort_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>6501</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LocalIP_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.20.209</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DataFile_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CC1_INPUT.txt</string>
   </property>
  </widget>
  <widget class="QLabel" name="DataFile_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DataFile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Prepare_pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>51</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="LocalPort_label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Local Port2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LocalPort_lineEdit_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>6501</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TargetPort_lineEdit_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>6501</string>
   </property>
  </widget>
  <widget class="QLabel" name="LocalIP_label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Local IP2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LocalIP_lineEdit_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.3.209</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TargetIP_lineEdit_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.3.201</string>
   </property>
  </widget>
  <widget class="QLabel" name="TargetIP_label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target IP2</string>
   </property>
  </widget>
  <widget class="QLabel" name="DataFile_label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DataFile2</string>
   </property>
  </widget>
  <widget class="QLabel" name="TargetPort_label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Port2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DataFile_lineEdit_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CC2_INPUT.txt</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>双网</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="UDPSendData.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Prepare_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>UDPSendDataClass</receiver>
   <slot>CreateSendThread()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>UDPSendDataClass</receiver>
   <slot>IfUseDoubleNet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CreateSendThread()</slot>
  <slot>IfUseDoubleNet()</slot>
 </slots>
</ui>
